types/webextension-polyfill/namespaces/sessions.d.ts

```diff
@@ -123,15 +118,15 @@ export namespace Sessions {
          * @param key The key which corresponds to the value being set.
          * @param value The value being set.
          */
-        setWindowValue(windowId: number, key: string, value: any): Promise<void>;
+        setWindowValue(windowId: number, key: string, value: unknown): Promise<void>;
 
         /**
          * Retrieve a value that was set for a given key on a given window.
          *
          * @param windowId The id of the window whose value is being retrieved from.
          * @param key The key which corresponds to the value.
          */
-        getWindowValue(windowId: number, key: string): Promise<any>;
+        getWindowValue(windowId: number, key: string): Promise<unknown>;
```

Comment:
This seems like another change from `any` to `unknown` in an output position.

Reply:
Same reason: only the caller knows what type it's supposed to be.